src/commands/azureServiceOperators/ui/azureservicebrowser.ts

```diff
@@ -16,21 +21,34 @@ export async function AzureServiceBrowser(explorer: k8s.ClusterExplorerV1): Prom
 }
 
 async function allServiceKinds(): Promise<AzureServiceKind[] | undefined> {
+    const apiResult = await getAPIResourceCommandResult();
+
+    if (apiResult.succeeded) {
+        return apiResult.result;
+    } else if (apiResult.error !== ASOInstallation.ASONotInstalled) {
```

Comment:
Consider dropping the `else` here and inserting a blank line above.  "If it's not the special error then display it, and in any case return undefined" is your error handling "paragraph"; the two arms of the current `if` statement address different concerns (return value vs user interaction).